src/commandDetails/coin/transfer.ts

```diff
@@ -0,0 +1,98 @@
+import { container } from '@sapphire/framework';
+import { User } from 'discord.js';
+import {
+  CodeyCommandDetails,
+  CodeyCommandOptionType,
+  CodeyCommandResponseType,
+  SapphireMessageExecuteType
+} from '../../codeyCommand';
+import { getCoinBalanceByUserId, changeDbCoinBalanceByUserId, UserCoinEvent } from '../../components/coin';
+import { getCoinEmoji, getEmojiByName } from '../../components/emojis';
+
+// Update coin balance of a user
+const coinUpdateExecuteCommand: SapphireMessageExecuteType = async (client, messageFromUser, args) => {
+  // First mandatory argument is user
+  const user = <User>args['user'];
+  if (!user) {
+    throw new Error('please enter a valid user mention or ID for balance adjustment.');
+  }
+
+  // Second mandatory argument is amount
+  const amount = args['amount'];
+  if (typeof amount !== 'number') {
+    throw new Error('please enter a valid amount to adjust.');
+  }
+
+  // Optional argument is reason
+  const reason = args['reason'] ?? null;
+
+  // Get balance
+  const senderBalance = await getCoinBalanceByUserId(messageFromUser.member!.user.id);
+  const recipientBalance = await getCoinBalanceByUserId(user.id);
+
+  // Implement some basic checks
+  if (messageFromUser.member!.user.id === user.id)
```

Comment:
Use curly braces - I think it looks cleaner

Reply:
Done.